target/linux/lantiq/patches-5.15/0747-net-dsa-lantiq_gswip-Update-per-port-flooding-and-le.patch

```diff
@@ -0,0 +1,33 @@
+From 87597566c5b598e85d9c31b713e0e1c98745cdfc Mon Sep 17 00:00:00 2001
+From: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
+Date: Thu, 28 Jul 2022 20:22:38 +0200
+Subject: [PATCH 747/768] net: dsa: lantiq_gswip: Update per port flooding and
+ learning defaults
+
+Disable flooding and learning by default. These are sane defaults for
+standalone (which the driver calls "single port bridge") ports. Apply
+these settings during setup and while disabling the port. This ensures
+that the settings are back to their defaults after removing a port from
+a bridge and using the same port in standalone mode instead (OpenWrt's
+netifd seems to leave flooding enabled when removing the port from a
+bridge).
+
+Signed-off-by: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
+[Do not disable flooding on CPU port in setup]
```

Comment:
In kernel 5.15 gswip_port_enable() is not called for the CPU port. Always enable flooding for the CPU port. 
Is this maybe helping? https://github.com/xdarklight/linux/commit/a595ee791bb57a816579c4ac321d0db2f8d155a8